src/App.tsx

```diff
@@ -1,9 +1,17 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
```

Comment:
according to the style guide, import styles should be at the bottom of the imports list

Reply:
corrected